plugins/filter_log_to_metrics/log_to_metrics.c

```diff
@@ -0,0 +1,792 @@
+/* -*- Mode: C; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
+
+/*  Fluent Bit
+ *  ==========
+ *  Copyright (C) 2015-2023 The Fluent Bit Authors
+ *  Copyright (C) 2023 SAP SE
```

Comment:
Companies cannot copyright code contributed to Fluent project (CNCF rule).

Reply:
Fixed.

Reply:
@edsiper (cc @patrick-stephens) Can you please review and approve the changes? Otherwise, merging is still blocked. Thanks!

Reply:
@edsiper (cc @patrick-stephens) Can you please review and approve the changes? Otherwise, merging is still blocked. Thanks!